packages/lib/services/spellChecker/SpellCheckerService.ts

```diff
@@ -115,7 +131,7 @@ export default class SpellCheckerService {
 		output.push({
 			label: _('Add to dictionary'),
 			click: () => {
-				void this.addToDictionary(this.language, misspelledWord);
+				void this.addToDictionary(this.languages[0], misspelledWord);
```

Comment:
Can't the list of languages be empty? In that case, shouldn't this call be skipped?

Reply:
It can, but not it this instance. There won't be any misspelled words if the languages array is empty. So this whole block of code will be skipped with the first line in the function `if (!misspelledWord) return [];`.